static/configeditor/index.html

```diff
@@ -722,14 +722,14 @@ <h3 id="docker-enable-outbound">Enabling outbound connections (optional)</h3>
 
   <div class="highlight" style="width:1000px;"><pre class="highlight shell tab-shell">
   <code style="font-size:smaller;">
-  com_ibm_watson_health_car_acd_host_1_label="Annotator_for_Clinical_Data_Container_Edition";\
-  com_ibm_watson_health_car_acd_host_1_url=https://&lt;your container edition url&gt;;\
-  com_ibm_watson_health_car_acd_host_1_proxy=wh-acd-ce;\
-  com_ibm_watson_health_car_acd_host_1_auth=Bearer;\
-  com_ibm_watson_health_car_acd_host_1_phi=false;\
+  clinical_data_annotator_car_acd_host_1_label="Annotator_for_Clinical_Data_Container_Edition";\
+  clinical_data_annotator_car_acd_host_1_url=https://&lt;your container edition url&gt;;\
+  clinical_data_annotator_car_acd_host_1_proxy=wh-acd-ce;\
```

Comment:
Are we still using wh- here as a prefix for the proxy? I guess this is just an example, but may want to get rid of that so we don't find it in searching for remaining work for blue rinse work items.